reporters_db/data/reporters.json

```diff
@@ -18316,15 +18382,27 @@
             "editions": {
                 "Rodney's Notes": {
                     "end": "1830-12-31T00:00:00",
+                    "regexes": [
+                        "$full_cite",
+                        "$volume_nominative ?$reporter, $page",
+                        "$reporter$page?"
```

Comment:
this looks wrong.

Reply:
by this ... I mean the lack of spacing between reporter and page.

Reply:
I was testing how regexes are generated with eyecite and i forgot to delete that.

Reply:
great.